tests/test_simsetup.py

```diff
@@ -1054,6 +1115,7 @@ def test_param_select_cross():
     assert uv_obj_cross == uv_obj_cross2
 
 
+@pytest.mark.filterwarnings("ignore:No beam information, so cannot determine")
```

Comment:
Probably we should update our test data to include beam info so we don't have to ignore these warnings.

Reply:
It's not a problem with the test data, we're explicitly turning off getting beam information in the call below (`return_beams=False`). The option to do this was something we added at your request, actually. But it does speed things up and this test doesn't need that stuff to be set, it's testing unrelated functionality.

Reply:
Ahh I see. Still, this feels wrong. We don't want a thousand warnings when you're doing something you want to (and are allowed to) do. Is there a way to catch this kind of circumstance so we don't raise a warning?

Reply:
I'm on the fence about this. I added this warning in this PR because for the first time there's a real consequence to the UVData metadata if you don't make the beams. Maybe the right thing to do is to add support for specifying this info in the telescope config file, in which case it can get it from there if (and only if) the beams aren't constructed?

Reply:
I think this is the right thing to do. If there is necessary uvdata metadata that can only be gotten from the beams, then we have to make a pathway to specifying that info outside specifying the beams.

Reply:
I started digging into this and I realized that with the new yaml constructors for the Analytic and UVBeam objects, the beams are basically always read in already, so there's not really a performance reason not to use the beams to set these parameters properly. And adding ways to pass them separately from the beams makes it possible for the information to get out of sync between the beams and the yaml file. So I think I'm going to change the code to use the beam information if it's available and only issue the warning if the beam info isn't available.

Reply:
Ok, I ended up halfway between. If you call `uvdata_init_from_params`, it gets the feed information from the beams specified in the telescope config file. If you call `uvdata_init_from_keywords` there are now options to pass the `feed_array`, `feed_angle`, and `mount_type` into that function.